browser/browsing_data/brave_browsing_data_remover_delegate.cc

```diff
@@ -71,8 +71,10 @@ void BraveBrowsingDataRemoverDelegate::RemoveEmbedderData(
 #if BUILDFLAG(ENABLE_BRAVE_NEWS)
   // Brave News feed cache
   if (remove_mask & chrome_browsing_data_remover::DATA_TYPE_HISTORY) {
-    brave_news::BraveNewsControllerFactory::GetForContext(profile_)
-        ->ClearHistory();
+    // auto controller = brave_news::BraveNewsController;
+    // brave_news::BraveNewsController::ClearHistory();
+    // brave_news::BraveNewsControllerFactory::GetForContext(profile_)
+    //     .ClearHistory();
```

Comment:
Is this intentionally commented out?

Reply:
no, looks like leftovers, but i don't remember why i would change that